src/components/ChartDetails.js

```diff
@@ -205,7 +205,9 @@ export class ChartDetails extends React.Component {
             <ReactMarkdown source={this.props.charts.spec.categoryDescription} />         
           </div>
           <div className = "tutorial-video-container">
-            {this.props.video!="" && this.props.charts.experiments===null ? (<button className ="tutorial-button" onClick={this.handleVideoOpenModal}><span className = "tutorial-title">Experiment Demo</span> <img className="tutorial-img" src={process.env.PUBLIC_URL + '/icons/play-square.png'}/></button>):("")}
+            {this.props.video!="" && this.props.charts.experiments===null ? (<button className ="tutorial-button" onClick={this.handleVideoOpenModal}>
+              <span className = "tutorial-title">Experiment Demo</span> 
```

Comment:
@aditya109  can you change the color of Experiment Demo to blue or make the text bold. 
cc: @chandankumar4